docs/training/apis-howto/api-pytorch-trainer-ug.rst

```diff
@@ -0,0 +1,354 @@
+       def __init__(self, context: PyTorchTrialContext, hparams: Dict) -> None:
+           self.context = context
+           self.model = context.wrap_model(nn.Sequential(
+               nn.Linear(9216, 128),
+           ))
+           self.optimizer = context.wrap_optimizer(torch.optim.Adadelta(
+               self.model.parameters(), lr=hparams["lr"])
+           )
+
+       def train_batch(
+               self, batch: pytorch.TorchData, epoch_idx: int, batch_idx: int
+       ) -> Dict[str, torch.Tensor]:
+           ...
+           output = self.model(data)
+           loss = torch.nn.functional.nll_loss(output, labels)
+
+           self.context.backward(loss)
+           self.context.step_optimizer(self.optimizer)
+
+           return {"loss": loss}
+
+       def evaluate_batch(self, batch: pytorch.TorchData) -> Dict[str, Any]:
+           ...
+           return {"validation_loss": validation_loss, "accuracy": accuracy}
+
+       def build_training_data_loader(self) -> DataLoader:
+           ...
+           return DataLoader(train_set)
+
+       def build_validation_data_loader(self) -> DataLoader:
+           ...
+           return DataLoader(validation_set)
+
+******************************************************
+ Step 2: Initialize the PyTorch Trial and the Trainer
+******************************************************
+
+After defining the PyTorch Trial, initialize the trial and the trainer.
+
+   Example:
+
+   ..
+      code::python
+
+      class MyUnion(schemas.UnionBase):
+          _id = "..."
+          _union_key = "type"
+
+      @MyUnion.member("a")
+      class MemberA(MyUnion):
+          _id = "..."
+
+.. code::
+
+   from determined import pytorch
+
+   def main():
+       # pytorch.init() returns a PyTorchTrialContext for instantiating PyTorchTrial
+       with det.pytorch.init() as train_context:
+           trial = MyPyTorchTrial(train_context)
+           trainer = det.pytorch.Trainer(trial, train_context)
+
+           # (Optional) Configure Determined profiler before calling .fit()
+           trainer.configure_profiler(enabled=True,
+                                      sync_timings=True,
+                                      begin_on_batch=0,
+                                      end_after_batch=10)
+
+           # Train
+           trainer.fit(
+               checkpoint_period=pytorch.Batch(10),
+               validation_period=pytorch.Batch(10),
+           )
+
+   if __name__ == "__main__":
+       # Configure logging here instead of through the expconf
+       logging.basicConfig(level=logging.INFO, format=det.LOG_FORMAT)
+       main()
+
+******************************************
+ Step 3: Run Your Training Script Locally
+******************************************
+
+With the PyTorch Trainer API, you can run training scripts locally without needing to use an
+experiment configuration (YAML) file. Be sure to specify max_length in the .fit() call, and
```

Comment:
`.fit()` needs the double backticks so it is formatted properly.  Also other variable names and method calls in this same sentence.